### DISCUSSION TITLE
Compound unique

### DISCUSSION THREAD

**Discussion by ff-anthony-soto** [prisma/prisma#2129] [Q&A] [answered]
Recently I updated to the latest version of prisma2.

But I get an error on my schema.

<img width="305" alt="Screen Shot 2020-04-07 at 5 38 44 PM" src="https://user-images.githubusercontent.com/59031228/78729276-e9526300-78f6-11ea-95d1-1281eea0034d.png">

<img width="1258" alt="Screen Shot 2020-04-07 at 5 41 28 PM" src="https://user-images.githubusercontent.com/59031228/78729352-09822200-78f7-11ea-8326-b29416b7c958.png">

Do you guys know the issue behind this?

Thanks!

**Comment by Sytten** [comment, marked as answer]
As per the release note, there is a new relation system in place and you need to update your existing relations to use a scalar field. Those fields can then be used to create the unique key.

**Reply by pantharshit00** [reply]
@ff-anthony-soto You can find the notes here: https://github.com/prisma/prisma/releases/tag/2.0.0-beta.1

**Reply by ff-anthony-soto** [reply]
Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
